relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -600,6 +611,41 @@ fn extract_span_metrics(
     Ok(())
 }
 
+static SQL_TABLE_EXTRACTOR_REGEX: Lazy<Regex> =
+    Lazy::new(|| Regex::new(r#"(?i)from(\s|\()+(?P<table>(\w+(\.\w+)*))(\s|\))+"#).unwrap());
```

Comment:
Should we request whitespace or a newline before `from`? For example,

```rust
INSERT INTO from_date VALUES (1, 2, 3);
```

should not match the regex.

I would add a test for this to be robust against regressions.

Reply:
It matches correctly, see [this test](https://github.com/getsentry/relay/pull/2142/commits/6c918bb9673ae78fae1298685bef2df3e7df0f2b#diff-decea315cf714306f36169f6d773ba8a05f2f2fc841a7684da0c52968c9337c7R885).